cmd/main.go

```diff
@@ -171,26 +176,35 @@ func main() {
 		panic(err)
 	}
 
-	diff := ""
+	diff := diff(*mat, ssComMat)
+
+	err = os.WriteFile(filepath.Join(destDir, "matrix-diff-ss"),
+		[]byte(diff),
+		0644)
+	if err != nil {
+		panic(err)
+	}
+}
+
+func diff(mat types.ComMatrix, ssMat types.ComMatrix) string {
```

Comment:
the name can be changed to something clearer, also is ssMat still correct here? can this just be mat1,mat2?